api/config.py

```diff
@@ -96,6 +98,15 @@ def apply_env_variables(config):
 ).get_default_database()
 log.debug(str(db))
 
+log_db = pymongo.MongoClient(
+    __config['persistent']['db_log_uri'],
+    j=True, # Requests only return once write has hit the DB journal
+    connectTimeoutMS=__config['persistent']['db_connect_timeout'],
+    serverSelectionTimeoutMS=__config['persistent']['db_server_selection_timeout'],
+    connect=False, # Connect on first operation to avoid multi-threading related errors
+).get_default_database()
+log.debug(str(log_db))
```

Comment:
I was following what we had above for the other MongoClient connection. It does help in dev to ensure that the connections were created properly.